lib/matplotlib/transforms.py

```diff
@@ -955,62 +897,62 @@ def update_from_data_xy(self, xy, ignore=None, updatex=True, updatey=True):
     def _set_x0(self, val):
         self._points[0, 0] = val
         self.invalidate()
-    x0 = property(BboxBase._get_x0, _set_x0)
+    x0 = property(BboxBase.x0.fget, _set_x0)
```

Comment:
No, because this would update the setter of `BboxBase` *too*.  I could write
```
@partial(property, BboxBase.x0.fget)
def x0(self, val): ...
```
but that hardly seems like an improvement in readability...